_el/exercises.md

```diff
@@ -336,172 +204,52 @@ piece of code with the output it produces".
+
+Η *Σύνοψη* απαιτεί επίσης από τους εκπαιδευόμενους να χρησιμοποιούν συλλογιστική υψηλότερης τάξης, και τους δίνει την ευκαιρία να εξασκήσουν μια δεξιότητα που είναι πολύ χρήσιμη όταν στοχεύει περισσότερη στην ανίχνευση σφαλμάτων παρά στη διόρθωσή τους. Για παράδειγμα, η ερώτηση προς τους μαθητές μπορεί να είναι: "Ποια πρόταση περιγράφει καλύτερα τον τρόπο με τον οποίο η έξοδος της f αλλάζει όταν το x παίρνει τιμές μεταξύ 0 και 10;" και δίνοντάς τους μετά πολλές επιλογές ως ερώτηση πολλαπλών επιλογών.
+
+Μπορείτε επίσης να ζητήσετε πολύ σύντομες απαντήσεις ελεύθερου κειμένου σε ερωτήσεις που έχουν πολύ στενό αντικείμενο, όπως για παράδειγμα "Ποιο είναι το βασικό χαρακτηριστικό ενός ευσταθούς αλγορίθμου ταξινόμησης;" Δεν μπορούμε ακόμα να αυτοματοποιήσουμε πλήρως τη βαθμολόγηση τέτοιων ερωτήσεων χωρίς ένα
+απογοητευτικό αριθμό ψευδών θετικών (αποδοχή λανθασμένων απαντήσεων) και ψευδών αρνητικών (απορρίπτοντας τα σωστά), αλλά ενδείκνυνται για αυτοαξιολόγηση ([s:individual-peer](#SECTION)).
+
+## Αυτόματη βαθμολόγηση {#s:exercises-grading}
+
+Τα αυτόματα εργαλεία βαθμολόγησης του προγράμματος υπάρχουν πολύ πριν γεννγθώ εγώ: η παλαιότερη δημοσιευμένη αναφορά χρονολογείται από το 1960 [[Holl1960](#CITE)], και οι έρευνες που δημοσιεύθηκαν στα [[Douc2005](#CITE),[Ihan2010](#CITE)] αναφέρουν ονομαστικά πολλά εργαλεία με αυτό το στόχο. Η κατασκευή τέτοιων εργαλείων είναι πολύ πιο πολύπλοκη από όσο μπορεί να φαίνεται αρχικά. Πως μπορούν να αντιπροσωπεύονται οι αναθέσεις; Πώς γίνεται η καταγραφή και παρακολούθησετι των απαντήσεων που υποβάλλονται; Μπορούν οι μαθητές να συνεργαστούν; Πώς μπορούν να γίνουν οι υποβολές απαντήσεων με ασφάλεια; [[Edwa2014a](#CITE)] είναι ένα άρθρο που αφιερώνεται εξ' ολοκλήρου σε ένα προσαρμοστικό σχήμα για την ανίχνευση και διαχείριση ατέρμονων βρόχων και άλλων μορφών κώδικα που δεν τερματίζουν κανονικά, και αυτό είναι μόνο ένα από τα πολλά ζητήματα που αναφέρονται.
+
+Όπως και σε άλλες περιπτώσεις, είναι σημαντικό να διακρίνει κανείς την ικανοποίηση που νοιώθουν οι μαθητές από τα μαθησιακά αποτελέσματα. [[Magu2018](#CITE)] σε ένα μάθημα Πληροφορικής δευτέρου έτους αντικατέστησαν τα ανεπίσημα εργαστήρια προγραμματισμού από ένα εβδομαδιαίο τεστ που αξιολογήθηκε από ένα εργαλείο αυτόματης βαθμολόγησης και το οποίο είχε αρχικά αναπτυχθεί για διαγωνισμούς προγραμματισμού. Παρόλο που στους μαθητές δεν άρεσε το το αυτοματοποιημένο σύστημα, το συνολικό ποσοστό αποτυχίας για το μάθημα μειώθηκε κατά το ήμισυ, και ο αριθμός των μαθητών που βαθμολογήθηκαν με άριστα τριπλασιάστηκε. Αντίθετα, οι [[Rubi2014](#CITE)] άρχισαν επίσης να χρησιμοποιούν ένα αυτοματοποιημένο σύστημα βαθμολόγησης σχεδιασμένο για διαγωνισμούς, αλλά δεν είδε σημαντική μείωση των ποσοστών εγκατάλειψης των μαθητών τους - για άλλη μια φορά, οι μαθητές έκαναν κάποια αρνητικά σχόλια σχετικά με το εργαλείο, τα οποία οι συγγραφείς αποδίδουν στα μηνύματά ανατροφοδότησης του ίδιου του εργαλείου παρά στην αντιπάθεια του εργαλείου καθ' αυτού.
```

Comment:
[[Magu2018](#CITE)]--> Οι [[Magu2018](#CITE)]
δεν ειδε--> δεν ειδαν

Reply:
Τhanks @pdelias ! Fixed in a new commit.